<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerVarVisualizerWidgetForm</class>
 <widget class="QWidget" name="SeerVarVisualizerWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QToolButton" name="expandAllToolButton">
     <property name="toolTip">
      <string>Expand all items below current selection.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/seer/resources/RelaxLightIcons/list-add.svg</normaloff>:/seer/resources/RelaxLightIcons/list-add.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="refreshToolButton">
     <property name="toolTip">
      <string>Refresh the list.</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/seer/resources/RelaxLightIcons/view-refresh.svg</normaloff>:/seer/resources/RelaxLightIcons/view-refresh.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="collapseAllToolButton">
     <property name="toolTip">
      <string>Collapse all items below current selection.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="9">
    <widget class="QIndexTreeWidget" name="variableTreeWidget">
     <column>
      <property name="text">
       <string>Variable</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>VarObj</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Exp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NumChild</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ThreadID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>HasMore</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QHistoryLineEdit" name="variableNameLineEdit">
     <property name="toolTip">
      <string>Enter a variable to visualize.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Variable name</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QCheckBox" name="debugCheckBox">
     <property name="toolTip">
      <string>Show more variable details.</string>
     </property>
     <property name="text">
      <string>Debug</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QCheckBox" name="autoRefreshCheckBox">
     <property name="toolTip">
      <string>Automatically refresh after each stopping point.</string>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHistoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QHistoryLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QIndexTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">QIndexTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>variableNameLineEdit</tabstop>
  <tabstop>expandAllToolButton</tabstop>
  <tabstop>collapseAllToolButton</tabstop>
  <tabstop>refreshToolButton</tabstop>
  <tabstop>autoRefreshCheckBox</tabstop>
  <tabstop>debugCheckBox</tabstop>
  <tabstop>variableTreeWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
